nuxeo-drive-client/nxdrive/tests/test_engine_offline.py

```diff
@@ -0,0 +1,132 @@
+'''
+Created on 11-Jul-2016
+
+@author: arameshkumar
+
+Files/folders should not sync while engine is paused due to a network error. In this test case, mock is used to simulate a network error
+
+'''
+
+from common_unit_test import UnitTestCase
+import nxdrive
+from mock import patch
+from urllib2 import URLError
+from time import sleep
+from nxdrive.client.base_automation_client import BaseAutomationClient
+
+server_online = True
+original_execute = BaseAutomationClient.execute
+original_fetch_api = BaseAutomationClient.fetch_api
+
+
+class EngineOffLineTestCase(UnitTestCase):
```

Comment:
So far the the convention for the filename has been <jira-number>_testcase.py and for the class name, <jira-number>TestCase.